### DISCUSSION TITLE
rtc subseconds alternative

### DISCUSSION THREAD

**Discussion by whoppercrab** [Lora-net/LoRaMac-node#1384] [Q&A]
I am currently using STM32L151CBU6 MCU. This hardware does not support rtc subseconds function. Is there any solution with SW implementation?

**Comment by mluis1** [comment]
Yes, there is one.
The original platform used for this project used similar MCU as yours. This platform has been removed since then as it is no more produced.

The latest version of the RTC implementation is available on v4.4.1 tag, Please refer to:
https://github.com/Lora-net/LoRaMac-node/blob/v4.4.1/src/boards/LoRaMote/rtc-board.c

Please note that some adaptations maybe required as a lot of things have changed since v4.4.1 release,

**Reply by whoppercrab** [reply]
Thanks to you, I'm implementing it using good references. A problem occurred during implementation. I'm having a problem with the timer interrupt handler not starting. Can you tell which part you are looking at?
